mitxpro/templates/footer.html

```diff
@@ -1,8 +1,32 @@
 {% load static %}
 
 <footer class="footer">
+    <div class="footer-top">
+        <div class="container">
+            <div class="row">
+                <div class="col-sm-6 clearfix">
+                    <img src={% static 'images/help.png' %}>
+                    <div class="info">
+                        <a href="https://xpro.zendesk.com" target="_blank">
```

Comment:
not sure, may be configurable these zendesk urls or go with that.

Reply:
I guess, these should not be related to wagtail.

Reply:
The zendesk URL doesn't need to be configurable. We can have only one.